draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -542,12 +543,12 @@ Some HTTPS client implementations perform real time third party checks of the
 revocation status of the certificates being used by TLS. If this check
 is done as part of the DNS API server connection procedure and the
 check itself requires DNS resolution to connect to the third party a
-deadlock can occur. The use of an OCSP {{RFC6960}} server is one
-example of how this deadlock can happen.
-To avoid the possibility of deadlock, DNS API servers SHOULD
+deadlock can occur. The use of OCSP {{RFC6960}} servers or AIA for CRL fetching
+({{RFC5280}} Section 4.2.2.1) are examples of how this deadlock can happen.
+To mitigate the possibility of deadlock, DNS API servers SHOULD
 bundle the certificate status as part of the handshake using a
 mechanism appropriate to the version of TLS, such as 
-using {{RFC6961}} for TLS version 1.2.
+using {{RFC6066}} Section 8 for TLS version 1.2.
```

Comment:
For AIA, you have to advise servers to provide intermediates that might otherwise be obtained by AIA chasing.

Reply:
see 84bfb0203fbcaaf97198d42353c0c2accea9b0c6 .. i appreciate the help here. further exactitude might require a pr.

Reply:
84bfb02 is good.  No further precision needed.